src/pathfinding_service/cli.py

```diff
@@ -59,6 +59,12 @@
     type=str,
     help="Place for a personal message to the customers",
 )
+@click.option(
+    "--matrix-server",
+    type=str,
+    multiple=True,
+    help="Use this matrix server instead of the default ones. Include protocol in argument.",
```

Comment:
Should we mention that multiple values are possible?

Reply:
I thought about it but also wanted to keep the help text concise. Since we don't even know a single case where we want to pass multiple servers, I removed it. Not sure if that was the right call. The cli help does not indicate the option of repeating that parameter, unfortunately.